Sources/BijectiveDictionary/BijectiveDictionary+Initializers.swift

```diff
@@ -44,6 +44,47 @@ extension BijectiveDictionary {
         self._rtl = Dictionary(uniqueKeysWithValues: reversePairs)
     }
     
+    /// Creates a new dictionary from the left-right pairs in the given sequence, using a combining
+    /// to resolve conflicts.
+    /// - Parameters:
+    ///   - pairs: A sequence of left-right pairs to use for the new dictionary.
+    ///   - combine: A closure that is called when a conflict is encountered, receiving the left-right pair
+    ///     and a description of the conflict that exists.
+    /// - Note: If the combine closure does not resolve the conflict for a given left-right pair, then that left-right
+    ///   pair will be excluded from the new dictionary.
+    @inlinable public init<S>(
+        _ pairs: S,
+        uniquingWith combine: (Element, Conflict) throws -> Element
```

Comment:
I thought the shape of `combine` was going to be `(Element, Element) throws -> Element` just like [Dictionary](https://developer.apple.com/documentation/swift/dictionary/init(_:uniquingkeyswith:)). 

I think that this could solve a lot of problems. It would be much closer to the Dictionary API. We would no longer be responsible for resolving the conflict. 

In fact, the shape of `combine` should probably be `(Element, Element, Conflict) throws -> Element` 

Then the user could `switch` on the `Conflict` and they could decide how to resolve it.

Reply:
After thinking some more, I think this approach could be good, but it still wouldn't **guarantee** that the conflict is resolved. The user would hand us an `Element` back, but they hand back a conflicting `Element` then we still have to decide what to do. 

Maybe we could define a `Resolution` type that the user could use to explicitly tell us how they want us to resolve the conflict. Then there's no ambiguity and we simply follow their directions. 

Then the shape of `combine` could be `(Element, Element, Conflict) throws -> (Element, Resolution)`. I'm not a fan of how complicated that closure shape would be, but I think it would solve our problems.